app/livechat/server/sendMessageBySMS.js

```diff
@@ -18,6 +19,8 @@ callbacks.add('afterSaveMessage', function(message, room) {
 		return message;
 	}
 
+	FileUpload.addJWTToFileUrlIfNecessary(message);
```

Comment:
As I commented before, IMO it would be better having a specific helper method to handle this process, and inside that method, I would call the addJWTToFileUrlIfNecessary method.
In that case, we could rename the addJWTToFileUrlIfNecessary to addJWTToFileUrl.